main.css

```diff
@@ -51,6 +68,7 @@ body {
     padding: 0;
     font-family: "Inter",sans-serif;
     user-select: none;
```

Comment:
Don't do this. It affects your site's accessibility. Let the user select.

Reply:
didn't know this, I'll remove this line of code. Thanks for pointing it out.